win32ss/user/ntuser/focus.c

```diff
@@ -341,6 +341,11 @@ IntDeactivateWindow(PTHREADINFO pti, HANDLE tid)
       UserDerefObjectCo(pwndFocus);
    }
 
+   //
+   // Check for keyboard modifiers and release them (CORE-14768)
+   //
```

Comment:
```suggestion
   // Check for keyboard modifiers and release them (CORE-14768)
```
Nitpick: I can see that the surrounding code is doing the same LOC-wasting style of comments already, yes. Nevertheless I do think that at some point in the future this will be subject of a change and we shouldn't add more to it therefore.